runtime/drivers/postgres/postgres.go

```diff
@@ -15,6 +15,37 @@ import (
 
 func init() {
 	drivers.Register("postgres", driver{})
+	drivers.RegisterAsConnector("postgres", driver{})
+}
+
+var spec = drivers.Spec{
+	DisplayName: "Postgres",
+	Description: "Connect to Postgres.",
+	SourceProperties: []drivers.PropertySchema{
+		{
+			Key:         "sql",
+			Type:        drivers.StringPropertyType,
+			Required:    true,
+			DisplayName: "SQL",
+			Description: "Query to extract data from Postgres.",
+			Placeholder: "select * from table;",
+		},
+		{
+			Key:         "pg_database_url",
+			DisplayName: "Postgress Connection String",
+			Type:        drivers.StringPropertyType,
+			Required:    false,
+			Href:        "https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING",
+			Placeholder: "postgresql://user:password@localhost/mydb",
```

Comment:
Use the default localhost connection string instead – IIRC it's `postgresql://postgres@localhost:5432/postgres` (but please double check)

Reply:
I think there is a password as well so I can use `postgresql://postgres:postgres@localhost:5432/postgres`